### PR TITLE
Fixed  UI  for remote nodes

### PR DESCRIPTION
**Pull request by AlexFront2021**
Fixed the remote node field instead of empty field shows trunked pubkey 
image

### CODE REVIEW

File: src/Pages/ChannelRequests.razor [hunk 1]
```diff
@@ -78,9 +79,21 @@
...
                             <ValidationError/>
                         </Addons>
-                        <FieldHelp>@(_selectedDestNode == null ? "Node Not found" : $"Found Node: {_selectedDestNode.Name}") </FieldHelp>
-                    </Validation>
-                </EditTemplate>
+               <FieldHelp>
+                    @{
+                        if (_selectedDestNode == null)
+                        {
+                            @("Node Not found");
```
Comment (Jossec101) [inline src/Pages/ChannelRequests.razor:86]: You can use html directly here, no need for a razor code.

E.g.:

<p> Node not found </p>

File: src/Pages/ChannelRequests.razor [hunk 2]
```diff
@@ -259,7 +272,21 @@
     <ChildContent>
         <DataGridColumn TItem="ChannelOperationRequest" Field="@nameof(ChannelOperationRequest.Id)" Caption="#" Sortable="false" Displayable="true"/>
         <DataGridColumn TItem="ChannelOperationRequest" Field="SourceNode.Name" Caption="Source Node" Sortable="false" Displayable="@IsAllRequestsColumnVisible(AllChannelsColumnName.SourceNode)"/>
-        <DataGridColumn TItem="ChannelOperationRequest" Field="DestNode.Name" Caption="Remote Node" Sortable="false" Displayable="@IsAllRequestsColumnVisible(AllChannelsColumnName.RemoteNode)"/>
+        <DataGridColumn TItem="ChannelOperationRequest" Field="DestNode.Name" Caption="Remote Node" Sortable="false" Displayable="true">
+            <DisplayTemplate>
+                @{
+                    string remoteNodeName = context.DestNode.Name;
```
Comment (Jossec101) [inline src/Pages/ChannelRequests.razor:278]:
```suggestion
                    var remoteNodeName = context.DestNode.Name;
```

File: src/Pages/ChannelRequests.razor [hunk 3]
```diff
@@ -78,9 +79,21 @@
                             </Addon>
                             <ValidationError/>
                         </Addons>
-                        <FieldHelp>@(_selectedDestNode == null ? "Node Not found" : $"Found Node: {_selectedDestNode.Name}") </FieldHelp>
-                    </Validation>
-                </EditTemplate>
+               <FieldHelp>
+                    @{
+                        if (_selectedDestNode == null)
```
Comment (RodriFS) [inline src/Pages/ChannelRequests.razor:84]: also you can do
```
@if (...)
{
   ...
}
```
instead of.
```
@{
   if(...)
   {
      ...
   }
}
```

Comment (AlexFront2021) [reply src/Pages/ChannelRequests.razor:84]: ok

**Review by RodriFS — APPROVED** [review]

**Pull request merged**